packages/dbos-cloud/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@dbos-inc/dbos-cloud",
+  "version": "0.0.0-placeholder",
+  "description": "Tool for performing application deployment to DBOS cloud",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/dbos-inc/dbos-ts",
+    "directory": "packages/dbos-cloud"
+  },
+  "main": "cli.ts",
+  "homepage": "https://docs.dbos.dev/",
+  "scripts": {
+    "build": "tsc --project tsconfig.json && chmod +x ./dist/packages/dbos-cloud/cli.js",
+    "test": "echo \"No tests\"",
```

Comment:
We will have to fix that at some point 😂

Reply:
In seriousness, if the tests aren't run out of here because they are functional tests, we could just make a note of that here?

Reply:
Mostly this CLI is tested through integration tests in the cloud. But there are bits we should eventually test with unit tests. The current note is fine!